lib/app/pages/profile/blocs/profile.dart

```diff
@@ -75,13 +92,51 @@ class ProfileBloc extends Bloc<ProfileEvent, ProfileState> {
     }
   }
 
+  void _submit(
+    ProfileDataEventSubmit event,
+    Emitter<ProfileState> emit,
+  ) async {
+    final log = Logger('talao-wallet/profile/submit');
+    try {
+      final firstName = event.model.firstName;
+      final lastName = event.model.lastName;
+      final phone = event.model.phone;
+      final location = event.model.location;
+      final email = event.model.email;
+
+      final key = (await SecureStorageProvider.instance.get('key'))!;
+      final did = DIDKitProvider.instance.keyToDID('key', key);
+      final verificationMethod = DIDKitProvider.instance.keyToVerificationMethod('key', key);
+      final options = {
+        'proofPurpose': 'assertionMethod',
+        'verificationMethod': verificationMethod
+      };
+
+      final selfIssued = SelfIssued('', 'SelfIssued', location, lastName,
+          firstName, phone, email, Author('', ''), did);
+
+      final credential = selfIssued.toJson();
+
+      final vc = await CreateCredential(
```

Comment:
Currently you're sending just the credential Subject. You need to create the whole credential which is containing je credential subject.